generate_secureblue_iso.sh

```diff
@@ -113,4 +113,5 @@ if is_yes "$generate_iso"; then
     $command
     mv deploy.iso $image_name.iso
     mv deploy.iso-CHECKSUM $image_name.iso-CHECKSUM
+    eval "sed -i 's/deploy.iso/'$image_name'.iso/' $image_name.iso-CHECKSUM"
```

Comment:
is there a way to do this without eval?

maybe this? `sed -i "s/deploy.iso/$image_name.iso/" "$image_name.iso-CHECKSUM"`